### PR TITLE
Module info check

### PR DESCRIPTION
**Pull request by zdenek-jonas**
Enforce plugin to check the presence of module-info.java.

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -352,6 +352,45 @@
 				</plugins>
 			</build>
 		</profile>
+		<profile>
+			<id>module-info-check</id>
+			<activation>
+				<file>
```
Comment (fh-ms) [inline pom.xml:358]: maybe `src/main/java` would be better than `target/classes`

File: pom.xml [hunk 2]
```diff
@@ -352,6 +352,45 @@
...
+						<artifactId>maven-enforcer-plugin</artifactId>
+						<version>3.0.0</version>
+						<executions>
+							<execution>
+								<id>enforce-files-exist</id>
+								<goals>
+									<goal>enforce</goal>
+								</goals>
+								<configuration>
+									<rules>
```
Comment (fh-ms) [inline pom.xml:375]: A better error message would be more helpful

**Review by fh-ms — CHANGES_REQUESTED** [review]

File: pom.xml (hunk 1 above)
Comment (zdenek-jonas) [reply pom.xml:358]: ok, changed

File: pom.xml (hunk 2 above)
Comment (zdenek-jonas) [reply pom.xml:375]: Ok, there were two checks, I have changed so, there is now one check only with the appropriate message.

**Review by fh-ms — APPROVED** [review]

**Pull request merged**